src/Nethermind/Nethermind.Core.Test/Caching/LruCacheTests.cs

```diff
@@ -172,5 +172,17 @@ public void Delete_keeps_internal_structure()
 
             count.Should().Be(itemsToKeep);
         }
+
+        [Test]
+        public void Wrong_capacity_number_at_constructor()
+        {
+            int maxCapacity = 0;
```

Comment:
but we are missing the other case test -> a regression test would be good that shows that previously above int numbers did not work?

Reply:
Sure, you are right, I will add it ASAP